src/util.rs

```diff
@@ -17,7 +17,7 @@
 use std::u64;
 
 use crate::eraftpb::{ConfChange, ConfChangeType, ConfState, Entry, Message};
-use protobuf;
+use protobuf::Message as Msg;
```

Comment:
This `use` appears in a lot of places, I think it is not necesary in most places to alias it to `Msg`, and it is better not to do that if we can

Reply:
This is necessary for using `protobuf`'s methods like `new()`.
Is it possible to drop this `use` (apart from completely migrating to prost)?

Reply:
I think we should not use `protobuf::Message::new`and instead use the inherent `new_` method from the wrapper. If we do need the `Message` trait for other things then it's fine to keep the import, but we should drop `as Msg`.